pkg/wdk/constants.go

```diff
@@ -3,4 +3,9 @@ package wdk
 const (
 	// BasketNameForChange is the name of the output basket that is used to store "change" outputs
 	BasketNameForChange = "default"
+
+	// PubKeyBytesLenCompressed is the length of the compressed pub key
+	PubKeyBytesLenCompressed = 33
+	// PubKeyBytesLenUncompressed is the length of the uncompressed pub key
+	PubKeyBytesLenUncompressed = 65
```

Comment:
those values doesn't need to be public, and actually should be placed in the same file as the validation that use it.